docs/input.md

```diff
@@ -75,7 +75,7 @@ import { Input } from 'react-native-elements';
 - [`errorMessage`](#errormessage)
 - [`errorStyle`](#errorstyle)
 - [`errorProps`](#errorprops)
-- [`inputComponent`](#inputcomponent)
+- [`InputComponent`](#inputcomponent)
```

Comment:
Can you move this to the bottom of the list?  See how Button does it.

Reply:
moved to the bottom